.buildkite/commands/checkout-editorial-branch.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -eu
+
+# EDITORIAL_BRANCH is passed as an environment variable from fastlane to Buildkite
+#
+if [[ -z "${EDITORIAL_BRANCH}" ]]; then
```

Comment:
TIL about this approach to avoid the `set -u` from the shebang to fail the script at this point because of an unbound access. Neat :clap: